test/cpp/interop/interop_client.h

```diff
@@ -127,9 +127,9 @@ class InteropClient {
 
    private:
     ChannelCreationFunc channel_creation_func_;
-    std::unique_ptr<TestService::Stub> stub_;
+    absl::optional<std::unique_ptr<TestService::Stub>> stub_;
```

Comment:
There's no need for `absl::optional<>` for the stub or the channel, since we don't need to differentiate between nullptr and nullopt.

Reply:
Done